### ISSUE TITLE
How to prevent complaints about private roads

### ISSUE THREAD

**Issue by joostschouppe** [osmbe/community_planning#33]
- [x]  ask RouteYou to show that paths are actually private
- [ ]  RouteYou uses https://tile.thunderforest.com/neighbourhood/ , so we can ask there too!
- [ ] write about this on the wiki: https://wiki.openstreetmap.org/wiki/Why_we_won%27t_delete_roads_on_private_property
- [x] ask Osmand to show private access by default (suggested by midgard)

**Comment by pietervdvn** [comment]
I've sent this to Thunderforest:

Hello,

Due to the coronavirus, people starting hiking a lot more here in Belgium (which is very good!).
However, they also go over private paths, annoying the owners (which is not so good). These hikers then often point to their GPS-device and state that 'there is a path here'.
This in turn results in angry landowners discovering OpenStreetMap and contacting us (the Belgian OSM-community) - sometimes friendly, sometimes with a threat to involve lawyers or sometimes just deleting lots of paths...
We then always answer that we'll mark their paths as private, as that is the official way to do it.

However, one of those GPS-applications popular in Belgium is 'RouteYou', which uses the Thunderforest Neighbourhood-baselayer. However, the private paths are rendered the same as public paths there, adding to the confusion.

Would it be possible to apply a different render style for such paths with `access=no` or `access=private`, such as an extremely thin and nearly invisible line? Or a small 'forbidden' icon? Of maybe even dropping them all together?

That would make our lives a lot easier.


Kind regards,

**Comment by joostschouppe** [comment]
Well that was quick:

![afbeelding](https://user-images.githubusercontent.com/10122639/87794291-adf4f800-c846-11ea-8344-84f45c34036f.png)

**Comment by pietervdvn** [comment]
This is an addition by "routeyou" and only visible there. I didn't get any response from thunderforest yet.

**Comment by joostschouppe** [comment]
Yeah, I just saw they have two similar layers. I've messaged RouteYou about this.

**Comment by joostschouppe** [comment]
Additional idea: put an actual text label on private roads / private paths. We could think this through and then make an issue on https://github.com/gravitystorm/openstreetmap-carto

**Comment by pietervdvn** [comment]
Maybe we should also make a wikipage with the most common reasons, to copy-past from or to link directly.

**Comment by joostschouppe** [comment]
You mean like https://wiki.openstreetmap.org/wiki/Why_we_won%27t_delete_roads_on_private_property linked above? :)

**Comment by joostschouppe** [comment]
Contacted mapy.cz / windy maps, as they also do not render private roads differently

**Comment by joostschouppe** [comment]
AllTrails.com has a pretty nice rendering: 

![afbeelding](https://user-images.githubusercontent.com/10122639/125160621-3b2f0c00-e17e-11eb-92c5-4927d8e7bd68.png)
